app/assets/javascripts/map/components/registration-form.js

```diff
@@ -61,13 +70,15 @@ function RegistrationForm() {
             onchange: event => { data.questions[question.slug] = event.currentTarget.value },
           })
         }),
-        alert ? m('.sya-registration__message', { class: alert.status }, alert.message) : null,
+        alert ? m('.sya-registration__message.error', { class: alert.status }, alert.message) : null,
```

Comment:
Not all of our alerts are errors, this bit of code already should add the error class if you set the status attribute of the alert properly. The reason we do this is because also return alerts from the Rails server, and use them to populate this alert message.
```javascript
{ class: alert.status }
```